src/Catalyst.Core.Modules.Rpc.Server.Tests/UnitTests/AddFileToDfsRequestObserverTests.cs

```diff
@@ -47,7 +47,9 @@
 using TheDotNetLeague.MultiFormats.MultiHash;
 using Xunit;
 
-namespace Catalyst.Core.Lib.Tests.UnitTests.P2P.IO.Observers
+//@TODO should be in rpc module test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4377003)